docs/sphinx/installation/Install in Windows.rst

```diff
@@ -121,15 +121,11 @@ If build process is completed without any errors, the driver binary will be move
 
 **Build the compression tools**
 
-Method 1:
+Method:
 
    - Navigate to the chipsec_tools\compression directory   
-   - run the build.cmd
-
-Method 2:
-
-   - Download compression tools from https://github.com/tianocore/edk2-BaseTools-win32/archive/master.zip   
-   - Unzip the archive into the chipsec_tools/compression/bin directory
+   - Run `python setup.py build`
+   - Copy the `EfiCompressor.*` file from  build/lib.* to the root chipsec directory
```

Comment:
"Copy the `EfiCompressor.*` file(s) from..."?  Copy a single file?  (Just for clarity.)